sys/arm/broadcom/bcm2835/bcm2835_gpio.c

```diff
@@ -109,6 +115,8 @@ struct bcm_gpio_softc {
 	device_t		sc_busdev;
 	struct mtx		sc_mtx;
 	struct resource *	sc_res[BCM_GPIO_IRQS + 1];
+	bool			sc_is2711;
+	u_int			sc_maxpins;
```

Comment:
`sc_res` and `sc_bst`/`sc_bsh` are functionally related, so we shouldn't insert the new struct members between them.

I would suggest placing the new ones right above `sc_gpio_npins`.